src/lib.rs

```diff
@@ -129,8 +125,8 @@ impl Invariant {
 
     pub fn create_pool(
         &mut self,
-        token_0: ActorId,
-        token_1: ActorId,
+        token_x: ActorId,
```

Comment:
What is the reason for changing the names of these tokens? We usually use `token_0` and `token_1` when the order doesn't matter, as it will be determined when creating the `PoolKey`. In this case, `token_x` won't always be `token_x` either - could be reversed